sass/custom.scss

```diff
@@ -169,9 +169,27 @@ a.hds-button--supplementary {
 }
 
 // Form group
+
 // Sets whitespace according to HDS
 .hs-form-group {
-  margin-bottom: $spacing-layout-s;
+  &:not(:last-child) {
+    margin-bottom: $spacing-layout-s;
+  }
+}
+
+// Apply error colours
+.hs-has-error.hs-form-group {
+  color: $color-error;
+}
+
+// Tee registration page has a longer form, which is why the designer
```

Comment:
Very helpful comment!

Reply:
The README isn't complete, but it should work for setting up a local version of this theme:
https://github.com/City-of-Helsinki/helsinki-keycloak-theme/blob/master/README.md

It can take some time though--maybe up to 30min--maybe even more if there are quirks.